packages/docs/docs/dev/reference.mdx

```diff
@@ -7,180 +7,141 @@ category: For Developers
 
 ## Methods
```

Comment:
```suggestion
```

Should remove and bump everything down a tag to match the hooks reference which looks a lot cleaner

<img width="1200" alt="Screenshot 2023-11-22 at 5 15 15 PM" src="https://github.com/FuelLabs/fuels-wallet/assets/106365423/a616e12d-0f3f-48c0-a762-b22405f4ba5f">